doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -350,6 +350,14 @@ Printing Ltac2 tactics
    :cmd:`Print` can print defined Ltac2 tactics and can avoid printing
    other objects by using `Print Ltac2`.
 
+.. cmd:: Ltac2 Globalize @ltac2_expr
+
+   This command prints the result of resolving notations in the given expression.
```

Comment:
```suggestion
   Prints the result of resolving notations in the given expression.
```

We usually omit boilerplate phrases that would appear in hundreds of places such as "This command".  The doc already shows "Command" next to the syntax.